src/zope/app/rotterdam/__init__.py

```diff
@@ -17,18 +17,21 @@
 from zope.publisher.interfaces.browser import IBrowserRequest
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
 
+
 class rotterdam(IBrowserRequest):
     """Layer for registering Rotterdam-specific macros."""
 
+
 class Rotterdam(rotterdam, IDefaultBrowserLayer):
     """The ``Rotterdam`` skin.
 
     It is available via ``++skin++Rotterdam``.
     """
 
+
 # BBB 2006/02/18, to be removed after 12 months
```

Comment:
It is actually a lot later than 12 months...

Reply:
Removed in https://github.com/zopefoundation/zope.app.rotterdam/pull/3/commits/e71ae576bf3d59692a962540efa72ab6f1572d4a.